cmd/installer/pkg/install/install.go

```diff
@@ -79,35 +95,36 @@ func Install(ctx context.Context, p runtime.Platform, mode Mode, opts *Options)
 	cmdline.SetAll(p.KernelArgs().Strings())
 
 	// first defaults, then extra kernel args to allow extra kernel args to override defaults
-	if err = cmdline.AppendAll(kernel.DefaultArgs); err != nil {
+	if err := cmdline.AppendAll(kernel.DefaultArgs); err != nil {
 		return err
 	}
 
-	if opts.Board != constants.BoardNone {
```

Comment:
if we drop this, how would disk image creation work for Talos < 1.7?

Reply:
hmmm, i'll re-check

Reply:
Doesn't it use the installer from that specific version?

Reply:
Disk image creation works using `main` installer code.